common/config/rush/.pnpmfile.cjs

```diff
@@ -33,5 +33,10 @@ function readPackage(packageJson, context) {
     packageJson.peerDependencies['typescript'] = '*';
   }
 
+  if (packageJson.name === '@serverless-stack/core') {

```

Comment:
Were these intended to be devDependencies? Should we raise a PR against that project?

Reply:
No, the SST project was originally trying to be a full toolchain that invokes TypeScript, ESLint, ESBuild, Jest, etc.  Their new roadmap is get out of that business and only own the operations specific to AWS lambdas, so I expect these problematic dependencies to go away in a future release.